src/Common/tests/StaticTestGenerator/StaticTestGenerator.csproj

```diff
@@ -9,12 +8,14 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn />
     <NullableContextOptions>enable</NullableContextOptions>
+    <Configurations>Debug;Release</Configurations>
+  </PropertyGroup>
+  <PropertyGroup>
+    <ProjectGuid>{67648882-2D9D-451A-BAC2-D16DE96895C2}</ProjectGuid>
```

Comment:
I thought we didn't need Guid's in the project anymore?

Reply:
I see the generator still adds these.  I bet we could remove it.  Fodder for deletion!  Not this PR though.